addons/website/views/website_templates.xml

```diff
@@ -111,13 +111,16 @@
             </t>
         </t>
         <!-- `alternate`/`canonical` mainly useful to crawlers/bots/SEO tools, which test the website as public user -->
-        <t t-if="request and request.is_frontend_multilang and website and website.is_public_user()">
+        <t t-if="request and request.is_frontend_multilang">
```

Comment:
The comment has to be deleted too, if you remove the related `and website and website.is_public_user()` line.
I'd keep those actually, there no need to process this whole block of code (xml/rendering + python calls) for internal users. It doesn't cost much, does it?

As you prefer: keep both or remove both :+1:

Reply:
ok, let's keep both.